macaroonbakery/discharge.py

```diff
@@ -5,6 +5,7 @@
 
 import macaroonbakery as bakery
 import macaroonbakery.checkers as checkers
```

Comment:
why not just `from macaroonbakery import checkers, error`?

Reply:
I prefer not to import the whole parent module when we actually only want to import the submodule, which is why I prefer "import xxx.yyy.zzz as zzz" because it's more obviously that it's just aliasing a normal import to another name.

Also, I *thought* we were avoiding recursive imports (hence the explicit import of the error package) but it seems many of the bakery packages do recursive imports anyway. I'm not keen on that, as I think partially imported packages are awkward to reason about, but I'll lose the import for now.